pkg/apis/workloads/v1alpha1/console_types.go

```diff
@@ -41,9 +41,21 @@ type ConsoleSpec struct {
 type ConsoleStatus struct {
 	PodName    string       `json:"podName"`
 	ExpiryTime *metav1.Time `json:"expiryTime,omitempty"`
-	Phase      string       `json:"phase"`
+	Phase      ConsolePhase `json:"phase"`
 }
 
+type ConsolePhase string
+
+// These are valid phases for a console
+const (
+	// ConsolePending means the console has been created but its pod is not yet ready
+	ConsolePending ConsolePhase = "Pending"
+	// ConsoleRunning means the pod has started and is running
+	ConsoleRunning ConsolePhase = "Running"
+	// ConsoleStopped means the console has reached its timeout
```

Comment:
This comment isn't strictly true; it could also have been that the pod completed before the job timeout

Reply:
True!